src/redux/store.ts

```diff
@@ -10,20 +10,15 @@ export const initState: AppState = {
     credential: { email: '', password: '' },
     userInfo: {
       role: '',
-      companyName: '',
-      companyInfo: '',
-      address: '',
-    },
-    info: {
-      jobPosts: [],
-      skills: [],
```

Comment:
Put these back aswell.

Reply:
everything went wrong !!!

Reply:
Well, it's just 7 lines of code. ;) I hope lessons learned to day.